module_cheatsheets/scRNA-seq_cheatsheet.md

```diff
@@ -0,0 +1,71 @@
+| `ggplot2`            | [`geom_vline()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/geom_abline)                    | Vertical Line   | Adds a vertical reference line to a `ggplot` or  `qplot`                                                                               |
+| `ggplot2`            | [`theme_classic()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/ggtheme)                     | Classic Theme   | Displays `ggplot` or `qplot` without gridlines                                                                                         |
+| `ggplot2`            | [`facet_wrap()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/facet_wrap)                     | Facet Wrap      | Plots individual graphs using specified variables to subset the data                                                                   |
+| `ggplot2`            | [`ggtitle()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/labs)                              | GG Title        | Sets the title of a `ggplot`                         |                                                       
+
+<div style="page-break-after: always;"></div>
+
+### `scran`
+Read the `scran` package documentation [**here**](https://www.rdocumentation.org/packages/scran/versions/1.10.2). <br>
+A vignette on the usage of the `scran` package can be found [**here**](https://bioconductor.org/packages/devel/bioc/vignettes/scran/inst/doc/scran.html).
+
+| Library/Package      | Piece of Code                           | What it's called                                                              | What it does                                                                                                           |
+|----------------------|------------------------------------|------------------------------------------------|---------------------------------------------------------------------------------------------------|
+| `scran`        | [`rowData()`](https://bioconductor.org/packages/devel/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html)          | Row Data   | Extracts and stores gene-level metadata. This function is also available in the `scater` package.                                                                            |
+| `scran`                | [`quickCluster()`](https://www.rdocumentation.org/packages/scran/versions/1.0.3/topics/Quick%20clustering)                | Quick Clustering     | Groups similar cells in clusters based on rank correlations in gene expression                                          |
+| `scran`                | [`computeSumFactors()`](https://bioconductor.org/packages/devel/bioc/vignettes/scran/inst/doc/scran.html)                 | Compute Sum Factors        | Returns a numeric vector of computed sum factors for each cell cluster in the object.                              |
```

Comment:
Can we add a brief point about what purpose the sum factors serve? May want to get @cansav09 to do that or weigh in.

Reply:
Can also check out: https://osca.bioconductor.org/normalization.html#normalization-by-deconvolution